lib/sentry/config.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Sentry.Config do
```

Comment:
I'm wondering if we load the config once at boot|compile and then return a struct with all the values set? Or do we need to hit the Application env ets table every time?